incubator/java-openliberty/stack.yaml

```diff
@@ -1,5 +1,5 @@
 name: Open Liberty
-version: 0.1.2
+version: 0.1.3
```

Comment:
No longer the latest..of course might not be 0.1.4 either ultimately.

Reply:
I'll remember to bump this up to whatever the latest is when we merge.